app/controllers/comments_controller.rb

```diff
@@ -534,4 +534,12 @@ def redirect_to_all_comments(commentable, options = {})
                   :page => options[:page]
     end
   end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(
+      :pseud_id, :content, :name, :email, :edited_at, :unreviewed
```

Comment:
No problem at all - I generally only add these params from some test or manual testing unpermitted attributes error. However, your comment makes sense if it was set to `attr_protected` before, so I went ahead and removed the param and pushed to this branch. As long as the Travis suite still passes, we should be all set. Thanks